_posts/2016-12-04-contributing-blog-post-to-pulpproject.org.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Contributing a blog post to pulpproject.org
+layout: post
+author: Dennis Kliban
+---
+Quickstart Guide
+----------------
+
+1. Fork the pulpproject.org repository on GitHub.
+   [https://github.com/pulp/pulpproject.org/](https://github.com/pulp/pulpproject.org/)
+
+1. Clone your fork locally and checkout the `gh-pages` branch.
+
+1. Create a new file in the `_posts` directory. The name for the file should be in the following
+   format: `2016-11-30-contributing-blog-post-to-pulpproject.org.md`
+
+1. Write a blog post in the new file.
+
+1. Optionally, you can build the site locally and view your blog post. More information on building
+   the site can be found in the
+   [README](https://github.com/pulp/pulpproject.org/blob/gh-pages/README.md).
+
+1. Commit your changes to the `gh-pages` branch.
+
+1. Push your `gh-pages` branch to GitHub.
+
+1. Preview the blog at `https://<username>.github.io/pulpproject.org/blog/`.
+
+1. Create a Pull Request against the `gh-pages` branch of [pulproject.org
+   repository](https://github.com/pulp/pulpproject.org/).
+
+Writing the Blog Post
+---------------------
+
+Pulpproject.org is a [GitHub Pages](https://pages.github.com/) site. Each page is created using
+Markdown. Using [this cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+should help with formatting the text. Each blog post begins with the [YAML Front
+Matter](http://jekyllrb.com/docs/frontmatter/). Front Matter begins and ends with `---`. In
+between the dashes, the author specifies a `layout`, an `author`, and a `title`. `tags` can be
```

Comment:
Can a block quoted example of the frontmatter be shown here after the description of it?